src/composition/models/stacking.jl

```diff
@@ -95,6 +99,11 @@ When training a machine bound to such an instance:
   evaluation of the learners in the Stack while training. This is not for the
   evaluation of the Stack itself.
 
+- `cache`: Weither machines created in the learning network will cache data or not.
+
+- `acceleration`: A supported `AbstractResource` to define the training parallelization
+  mode of the learning network.
```

Comment:
Some users will use `Stack` but not be familiar with the term "learning network", so I suggest dropping the reference here. If they try an unsupported resource, they get an informative message explaining that. 

```suggestion
- `acceleration`: an `AbstractResource` defining the mode of parallelization for training the stack
```

Reply:
But we should also update this line: https://github.com/JuliaAI/MLJBase.jl/blob/37a5256fa85eec02d91e340b0f4c0a1ddcbbf44a/src/composition/learning_networks/nodes.jl#L210

to include `CPUThreads()` as a valid resource.